api.js

```diff
@@ -5,7 +5,26 @@ module.exports = {
   createTechOps: async function(payload) {
     return axios.post(process.env.API_URL + '/techops', payload, requestHeader())
   },
-  
+
+  createSalesforceCase: async function(payload, authResponse) {
+    return axios.post(
+      process.env.SERVICE_CLOUD_URI + "/api/v1/cases",
+      payload,
+      requestServiceCloudAppHeader(authResponse)
+    );
+  },
+
+  authenticateSalesforce: async function() {
+    let credentials_payload = `{ "username": "${process.env.SERVICE_CLOUD_USER}", "password": "${process.env.SERVICE_CLOUD_PASSWORD}"}`;
+    let header = { headers: { "Content-Type": "application/json" } };
```

Comment:
Como essa variável não está sendo alterada, podemos utilizar o `const`.

```suggestion
    const header = { headers: { "Content-Type": "application/json" } };
```